sparkmagic/sparkmagic/magics/sparkmagicsbase.py

```diff
@@ -42,6 +45,11 @@ def __init__(self, shell, data=None, spark_events=None):
         self.ipython_display = IpythonDisplay()
         self.spark_controller = SparkController(self.ipython_display)
 
+        self.default_spark_output_handler = SparkOutputHandler(
+                              handle_html=lambda out: self.ipython_display.html(out),
+                              handle_text=lambda out: self.ipython_display.write(out),
+                              handle_default=lambda out: self.ipython_display.display(out))
```

Comment:
You're right, the lambda is not necessary

Reply:
I remember now why I included the lambdas

[One of the kernel_magic tests swaps out the ipython_display](https://github.com/jupyter-incubator/sparkmagic/blob/master/sparkmagic/sparkmagic/tests/test_kernel_magics.py#L44):
```
    magic.ipython_display = ipython_display = MagicMock()
    ...
    ipython_display.write.assert_called_once_with(line)
```

If my constructor binds `handle_text` to the internally created `ipython_display.write`, then it will not use a reassigned  `ipython_display`  

If we use a lambda, it'll look up the value of the  `ipython_display`  when it's called